sw-services/src/main/java/org/egov/swservice/model/Connection.java

```diff
@@ -0,0 +1,578 @@
+@Validated
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2020-03-13T11:29:47.358+05:30[Asia/Kolkata]")
+public class Connection {
+	@JsonProperty("id")
+	private String id = null;
+
+	@JsonProperty("property")
+	private Property property = null;
+
+	@JsonProperty("applicationNo")
+	private String applicationNo = null;
+
+	/**
+	 * Gets or Sets applicationStatus
+	 */
+	public enum ApplicationStatusEnum {
+		INITIATED("INITIATED"),
+
+		REJECTED("REJECTED"),
+
+		PENDING_FOR_CITIZEN_ACTION("PENDING_FOR_CITIZEN_ACTION"),
+
+		PENDING_FOR_DOCUMENT_VERIFICATION("PENDING_FOR_DOCUMENT_VERIFICATION"),
+
+		PENDING_FOR_FIELD_INSPECTION("PENDING_FOR_FIELD_INSPECTION"),
+
+		PENDING_APPROVAL_FOR_CONNECTION("PENDING_APPROVAL_FOR_CONNECTION"),
+
+		PENDING_FOR_PAYMENT("PENDING_FOR_PAYMENT"),
+
+		PENDING_FOR_CONNECTION_ACTIVATION("PENDING_FOR_CONNECTION_ACTIVATION"),
+
+		CONNECTION_ACTIVATED("CONNECTION_ACTIVATED"),
+
+		APPLIED("Applied"),
+
+		APPROVED("Approved"),
+
+		CANCELED("Canceled"),
+
+		INPROGRESS("InProgress");
+
+		private String value;
+
+		ApplicationStatusEnum(String value) {
+			this.value = value;
+		}
+
+		@Override
+		@JsonValue
+		public String toString() {
+			return String.valueOf(value);
+		}
+
+		@JsonCreator
+		public static ApplicationStatusEnum fromValue(String text) {
+			for (ApplicationStatusEnum b : ApplicationStatusEnum.values()) {
+				if (String.valueOf(b.value).equals(text)) {
+					return b;
+				}
+			}
+			return null;
+		}
+	}
+
+	@JsonProperty("applicationStatus")
+	private ApplicationStatusEnum applicationStatus = null;
+
+	/**
+	 * Gets or Sets status
+	 */
+	public enum StatusEnum {
+		ACTIVE("Active"),
+
+		INACTIVE("Inactive"),
+
+		INWORKFLOW("INWORKFLOW");
+
+		private String value;
+
+		StatusEnum(String value) {
+			this.value = value;
+		}
+
+		@Override
+		@JsonValue
+		public String toString() {
+			return String.valueOf(value);
+		}
+
+		@JsonCreator
+		public static StatusEnum fromValue(String text) {
+			for (StatusEnum b : StatusEnum.values()) {
+				if (String.valueOf(b.value).equals(text)) {
+					return b;
+				}
+			}
+			return null;
+		}
+	}
+
+	@JsonProperty("status")
+	private StatusEnum status = null;
+
+	@JsonProperty("connectionNo")
+	private String connectionNo = null;
+
+	@JsonProperty("oldConnectionNo")
+	private String oldConnectionNo = null;
+
+	@JsonProperty("documents")
+	@Valid
+	private List<Document> documents = null;
+
+	@JsonProperty("plumberInfo")
+	@Valid
+	private List<PlumberInfo> plumberInfo = null;
+
+	@JsonProperty("roadType")
+	private String roadType = null;
+
+	@JsonProperty("roadCuttingArea")
+	private Float roadCuttingArea = null;
+
+	@JsonProperty("connectionExecutionDate")
+	private BigDecimal connectionExecutionDate = null;
+
+	@JsonProperty("connectionCategory")
+	private String connectionCategory = null;
+
+	@JsonProperty("connectionType")
```

Comment:
@NotNull notation should be present wherever NOT NULL constraint is there in db. Else data persistence will fail in persister